models/likesDao.js

```diff
@@ -0,0 +1,50 @@
+const dataSource = require("./dataSource");
+
+const likes = async (userId, foodId) => {
+  try {
+    const isExists = await dataSource.query(
+      `
+                SELECT EXISTS (
+                    SELECT * FROM likes 
+                    WHERE (
+                      user_id = ? AND food_id = ?
+                    )
+                )
+        `,
+      [userId, foodId]
+    );
+
+    if (Object.values(isExists[0]) == 1) {
```

Comment:
```suggestion
    const [ isExists ] = await dataSource.query(
      `
                SELECT EXISTS (
                    SELECT * FROM likes 
                    WHERE (
                      user_id = ? AND food_id = ?
                    )
                )
        `,
      [userId, foodId]
    );

    if (Object.values(isExists[0]) == 1) {
```

- 위와 같이 isExists를 한 번 대괄호로 감싸면 17번째 라인의 가독성을 조금 높일 수 있겠네요!